src/core/psxmem.cc

```diff
@@ -89,7 +89,17 @@ int PCSX::Memory::init() {
     m_readLUT = (uint8_t **)calloc(0x10000, sizeof(void *));
     m_writeLUT = (uint8_t **)calloc(0x10000, sizeof(void *));
 
-    m_wram = (uint8_t *)calloc(0x00800000, 1);
+    uint32_t pid = 0;
+#if defined(_WIN32) || defined(_WIN64)
+    pid = static_cast<uint32_t>(GetCurrentProcessId());
+#elif defined(__linux__) || (defined(__APPLE__) && defined(__MACH__))
+    pid = static_cast<uint32_t>(getpid());
+#endif
+
+    // Init all memory as named mappings
+    m_wramShared.init(_("wram"), 0x00800000);
```

Comment:
No, let's avoid making it "wram" a localized string. This would break external scripts depending on the selected locale.